### PR TITLE
ApproxTimeEvolution.compute_decomposition docstring has example

### PR DESCRIPTION
**Pull request by isaacdevlugt**
**Context:**

The docstring of `compute_decomposition` for `ApproxTimeEvolution` is slightly misleading with the `coeffs_and_time` argument.

**Description of the Change:**

Adds an example to the docstring to clear things up.

**Related GitHub Issues:**

https://discuss.pennylane.ai/t/approxtimeevo-decomposition-bug/3178/2

### CODE REVIEW

File: pennylane/templates/subroutines/approx_time_evolution.py
Comment (albi3ro) [inline pennylane/templates/subroutines/approx_time_evolution.py:1]: Could you update line 132 to say:
```
*coeffs_and_time (TensorLike): coefficients of the Hamiltonian, appended by the time
```

File: pennylane/templates/subroutines/approx_time_evolution.py [hunk 1]
```diff
@@ -140,6 +140,31 @@ def compute_decomposition(
...
+            import pennylane as qml
+            from pennylane.templates import ApproxTimeEvolution
+
+            num_qubits = 2
+
+            hamiltonian = qml.Hamiltonian(
+                [0.1, 0.2, 0.3], [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0), qml.PauliX(1)]
+            )
+
+            dev = qml.device("default.qubit", wires=num_qubits)
```
Comment (albi3ro) [inline pennylane/templates/subroutines/approx_time_evolution.py:156]: don't think you need a device here.

File: pennylane/templates/subroutines/approx_time_evolution.py [hunk 2]
```diff
@@ -140,6 +140,31 @@ def compute_decomposition(
 
         Returns:
             list[.Operator]: decomposition of the operator
+
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane.templates import ApproxTimeEvolution
```
Comment (albi3ro) [inline pennylane/templates/subroutines/approx_time_evolution.py:148]:
```suggestion
            from pennylane import ApproxTimeEvolution
```

**Review by timmysilv — APPROVED** [review]
aside from Christina's comments, the docs [lgtm](https://xanaduai-pennylane--4354.com.readthedocs.build/en/4354/code/api/pennylane.ApproxTimeEvolution.html#pennylane.ApproxTimeEvolution.compute_decomposition)! thanks for adding this 🎉

File: pennylane/templates/subroutines/approx_time_evolution.py (hunk 2 above)
Comment (isaacdevlugt) [reply pennylane/templates/subroutines/approx_time_evolution.py]: Was also in the docstring for the op itself. Removed there as well.

**Review by albi3ro — APPROVED** [review]
Thanks 👍

**Pull request merged**